tests/system/model_monitoring/test_model_monitoring.py

```diff
@@ -615,6 +614,18 @@ def test_model_monitoring_voting_ensemble(self):
         # it can take ~1 minute for the batch pod to finish running
         sleep(60)
 
+        # Validate that the KV schema has been generated as expected
+        client = get_v3io_client(endpoint=mlrun.mlconf.v3io_api)
+        cursor = client.kv.new_cursor(
```

Comment:
Not sure why we need to list the dir for it? Better just `get()` the file we are looking for. Even better, use `object.get()` and check the content while we're at it.

Reply:
Wasn't aware for the `object.get()` operation. I've updated the PR with this method + validating several fields within the schema.

Reply:
I did